Dockerfile

```diff
@@ -21,15 +21,15 @@ RUN echo "Cargo target: $CARGO_BUILD_TARGET"
 
 ADD config.toml /usr/local/cargo/
 # CARGO_BUILD_TARGET is respected by cargo install and other cargo commands
-RUN cargo install --root /cargo-cross cargo-chef@0.1.67 cargo-sbom@0.9.1
+RUN cargo install --root /cargo-cross cargo-chef@0.1.71 cargo-sbom@0.9.1
 
 
 FROM rust:${RUST_VERSION}-alpine3.20 AS tools-target-base
 ENV TARGETS="x86_64-unknown-linux-musl aarch64-unknown-linux-musl x86_64-unknown-linux-gnu aarch64-unknown-linux-gnu riscv64gc-unknown-linux-gnu"
 RUN rustup target add ${TARGETS}
 
 # needed for cargo-chef and cargo-sbom, as well as many other compilations
-RUN apk add musl-dev linux-headers make clang mold python3 git
+RUN apk add musl-dev linux-headers make clang mold python3 git perl protoc
```

Comment:
should we maybe just include it in the builder container for vector? or could other rust-based EVE parts benefit from this components?

Reply:
@europaul this is what I do now but we must not call `apk add` in the container this is why I add it here.

Reply:
but I thought we can include them in the "build" stage of vector's dockerfile

Reply:
@europaul we shouldn't. The container will be built under LK and the build will not be reproducible. Alpine can have version of the tool updated between builds.

Reply:
but you fix the version of alpine some place and then the packages' versions are also fixed. or am I wrong?